### PR TITLE
Feature/react examples

### PR DESCRIPTION
**Pull request by meovan07**
Examples React:
- Lifting state up example.
- Hooks example.
- Context example.
- Virtualize long list example.

### CODE REVIEW

File: react/examples/context-example/src/index.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-text-size-adjust: 100%;
+}
+
+a {
+  font-weight: 500;
+  color: #646cff;
+  text-decoration: inherit;
+}
```
Comment (GraphicDThanh) [inline react/examples/context-example/src/index.css:22]: enter

File: react/examples/storybook-example/src/components/button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+/** this is a special button!! */
+export function Button({
+  children,
+  variant = 'primary',
+}: ButtonProps): React.ReactElement {
+  return (
+    <button
+      type="button"
+      style={{
+        backgroundColor: variant === 'primary' ? 'blue' : 'gray',
```
Comment (GraphicDThanh) [inline react/examples/storybook-example/src/components/button/Button.tsx:21]: Define the value first will more redable, like:
```
const backgroundColor = variant === 'primary' ? 'blue' : 'gray'
```

then use
```
style={{
        backgroundColor: backgroundColor,
```

**Comment by GraphicDThanh** [comment]
Look good! 

Just some small things:
- delete .gitkeep if not needed anymore.
- re-check code style CSS, block CSS should have an empty line between
- still have some warnings(red circle) on code changes
- some comments on PR

Tips on view changes: Use the checkbox `Viewed` to collapse the file you have reviewed/seen

**Comment by meovan07** [comment]
> Look good!
> 
> Just some small things:
> 
> * delete .gitkeep if not needed anymore.
> * re-check code style CSS, block CSS should have an empty line between
> * still have some warnings(red circle) on code changes
> * some comments on PR
> 
> Tips on view changes: Use the checkbox `Viewed` to collapse the file you have reviewed/seen

Thanks for your comments, I got it. If have any issues or questions, I will let you know as soon as possible

**Review by GraphicDThanh — APPROVED** [review]
Great work! Keep it up on your next practice 👍

**Pull request merged**